packages/debugger/test/data/global.js

```diff
@@ -160,7 +160,13 @@ describe("Globally-available variables", function () {
     provider = Ganache.provider({
       seed: "debugger",
       gasLimit: 7000000,
-      _chainId: 1337 //temporary until Ganache v3!
+      hardfork: "istanbul",
```

Comment:
(Btw, my preferred approach for this would be to actually update this test to remove the hack, but uh I think it's fine to leave that in for now and I can remove it later. :P )

Reply:
David suggested that it is likely here because these tests might rely on `gasPrice = 0`.

Reply:
Oh, these tests don't rely on gas price being zero; they do, however, currently rely on base fee being zero (until the hack is removed and the tests are properly updated :) ).  So, like, if you're going to pin a specific hardfork, you could at least use Berlin rather than Instabul. :)

Reply:
Let's make sure we don't forget the follow-on here... maybe open an issue?

Reply:
Issue made here https://github.com/trufflesuite/truffle/issues/4633

Reply:
OK I went and updated this particular one to London; the rest are still on Istanbul though.